src/node.cc

```diff
@@ -3480,6 +3480,12 @@ static void PrintHelp() {
          "                           does not appear to be a terminal\n"
          "  -r, --require            module to preload (option can be "
          "repeated)\n"
+#if HAVE_INSPECTOR
+         "  --inspect[=host:port] activate inspector on host:port\n"
+         "                        (default: 127.0.0.1:9229)\n"
+         "  --inspect-brk[=host:port]  activate inspector on host:port\n"
+         "                             and break at start of user script\n"
```

Comment:
--inspect-port was omitted from the docs, see https://github.com/nodejs/node/pull/12581